labelbox/pagination.py

```diff
@@ -79,6 +79,32 @@ def __next__(self):
         self._data_ind += 1
         return rval
 
+    def get_one(self):
+        """Iterates over self and returns first value
+        This method is idempotent
+        """
+        for value in self:
+            return value
+
+    def get_many(self, n: int):
+        """Iterates over self and returns first n results
+        This method is idempotent
+
+        Args:
+            n (int): Number of elements to retrieve
+        """
+        results = []
+        i = 0
+
+        for value in self:
```

Comment:
This has the same issue as above. Where you might want the first 200 but if you call the function twice it will fetch result 200-400. Even though this function only returns results 0 to 200